content/faq/how-do-I-use-composer-with-github-access-token.md

```diff
@@ -0,0 +1,51 @@
+---
+title: How do I use Composer with GitHub personal access token?
+menu:
+  main:
+    parent: 'faq'
+    weight: 40
+weight: 40
+linkTitle: Setup GitHub Token
+---
+If you need to download private repositories from GitHub during the build
+process, an access token has to be available during `composer install`.
+
+To set up that, make sure you are using one of the Docker images from
```

Comment:
This is the documentation of ContinuousPipe, and CP's base images are not mandatory. I'd therefore replace this paragraph with something like:

> This FAQ is showing how to setup the GitHub token with an image based on [ContinuousPipe dockerfiles](https://github.com/continuouspipe/dockerfiles). These images are prepared to call `composer global config github-oauth.github.com "$GITHUB_TOKEN"` when there is a `GITHUB_TOKEN` environment variable.